### PR TITLE
added new Traffic Dashboards

### PR DESCRIPTION
**Pull request by DylanSchneider**

### CODE REVIEW

File: resources/dashboards/Traffic-Endpoints-Dashboard.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+   "hits": 0, 
+   "timeRestore": false, 
+   "description": "", 
+   "title": "Traffic Endpoints Dashboard", 
+   "panelsJSON": "[{\"col\":10,\"id\":\"Top-Applications-By-Sessions-(pie)\",\"row\":3,\"size_x\":3,\"size_y\":2,\"type\":\"visualization\"},{\"col\":10,\"id\":\"Top-Applications-By-Metadata-Impact-(pie)\",\"row\":1,\"size_x\":3,\"size_y\":2,\"type\":\"visualization\"},{\"col\":7,\"id\":\"Top-Applications-By-Bandwidth-(pie)\",\"row\":1,\"size_x\":3,\"size_y\":2,\"type\":\"visualization\"},{\"col\":7,\"id\":\"Top-Applications-By-Packet-Count\",\"row\":3,\"size_x\":3,\"size_y\":2,\"type\":\"visualization\"},{\"col\":1,\"id\":\"Top-Applications-By-Sessions-(histogram)\",\"row\":1,\"size_x\":6,\"size_y\":4,\"type\":\"visualization\"},{\"col\":7,\"id\":\"Sum-of-Repeated-Field-Count-by-Application\",\"row\":8,\"size_x\":6,\"size_y\":3,\"type\":\"visualization\"},{\"col\":1,\"id\":\"Average-Repeated-Elements-in-a-Message-(All-Reports)\",\"row\":5,\"size_x\":6,\"size_y\":3,\"type\":\"visualization\"},{\"id\":\"Average-Message-Size-(All-Reports)\",\"type\":\"visualization\",\"size_x\":6,\"size_y\":3,\"col\":7,\"row\":5}]",
```
Comment (craig-cogdill) [inline resources/dashboards/Traffic-Endpoints-Dashboard.json]: I know you didn't create this visualization, but `Top-Applications-By-Sessions-(histogram)` is just wrong. It's sorting by `Sum of BytesDelta`, so we should fix that while we're here too.

Comment (josh-cogdill) [inline resources/dashboards/Traffic-Endpoints-Dashboard.json]: Some of these visualizations have `(pie)` or `(histogram)` in the title and some do not. Let's make these consistent. It looks like other dashboards have them in the titles, so for those that do not lets add it.

File: resources/dashboards/Traffic-Profile-Dashboard.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+   "hits": 0, 
+   "timeRestore": false, 
+   "description": "", 
+   "title": "Traffic Profile Dashboard",
```
Comment (craig-cogdill) [inline resources/dashboards/Traffic-Profile-Dashboard.json]: I think the names of these two dashboards are switched...

This dashboard displays a table with Src & Dest IPs, as well as visualizations about the 'top talkers' - the Src and Dest IPs that account for the most sessions and highest metadata impact.

The other dashboard shows stats about the volume of traffic, the size of the average JSON insertion, etc.

The first description sounds like a dashboard centered around Endpoints, and the second description sounds like a dashboard centered around your traffic profile.

**Comment by craig-cogdill** [comment]
screen shot 2018-06-19 at 2 03 04 pm

Unintended side effect of renaming that visualization. I think that change should be reverted.

File: resources/dashboards/Traffic-Profile-Dashboard.json (hunk 2 above)
Comment (josh-cogdill) [reply resources/dashboards/Traffic-Profile-Dashboard.json]: Agreed @DylanSchneider

**Pull request merged**